### ISSUE TITLE
.dev file extension is used as version number

### ISSUE THREAD

**Issue by EyMaddis** [ds300/patch-package#284]
I have react-native in my devDependencies (since this is also a web app) and therefore renamed the patch file `react-native+0.62.2.dev.patch`.
However, this now leads to warnings everytime I install packages (yarn in my case).

```

Warning: patch-package detected a patch file version mismatch

  Don't worry! This is probably fine. The patch was still applied
  successfully. Here's the deets:

  Patch file created for

    react-native@0.62.2.dev

  applied to

    react-native@0.62.2
  
  At path
  
    node_modules/react-native

  This warning is just to give you a heads-up. There is a small chance of
  breakage even though the patch was applied successfully. Make sure the package
  still behaves like you expect (you wrote tests, right?) and then run

    patch-package react-native

  to update the version in the patch file name and make this warning go away.

```

It seems that patch-package thinks that .dev is part of the version number.

**Comment by EyMaddis** [comment]
Ahh... somebody already made a PR for that: https://github.com/ds300/patch-package/pull/224
I will close this ticket once this landed

**Comment by felixmosh** [comment]
Is it possible to review #224 pr?

**Comment by ds300** [comment]
This was fixed in [v6.3.0](https://github.com/ds300/patch-package/releases/tag/v6.3.0)
